game/scripts/npc/abilities/antimage_mana_break.txt

```diff
@@ -28,7 +28,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "mana_per_hit"                                    "28 40 52 64 88 124"
+        "mana_per_hit"                                    "28 40 52 64 88 160"
```

Comment:
Can the scaling on mana break be "96 128" at OAA levels and then we get antimages current base attack time of 1.4 instead of his nerfed BAT of 1.5 in OAA.

Reply:
You said it yourself that Anti-Mage is fine. Don't get influenced by Pigeon.